media/Config/autoCompletePath.js

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const outputToInput = function (toolIndex, nextInputValue) {
+  for (let k = toolIndex + 1; k < oneToolList.length; k++) {
+    if (oneToolList[k].use === true) {
+      for (let l = 0; l < oneToolList[k].options.length; l++) {
+        if (oneToolList[k].options[l].optionName === "input_path") {
+          oneToolList[k].options[l].optionValue = nextInputValue;
+          break;
+        }
+      }
+      break;
+    }
+  }
+};
+
+const makeOutputAuto = function (input, toolIndex, optionIndex) {
+  // because os maybe win32, divisor can be '\\'
+  let divisor = "/";
+  if (input.includes("\\")) {
+    divisor = "\\";
+  }
+  let paths = input.split(divisor);
+  let tmp = paths[paths.length - 1].split(".");
+  switch (oneToolList[toolIndex].type) {
+    case "one-optimize": {
+      tmp.splice(tmp.length - 1, 0, "opt");
```

Comment:
It would be helpful to let developers know about the name with comment.
```suggestion
      # ${filename}.* -> ${filename}.opt.*
      tmp.splice(tmp.length - 1, 0, "opt");
```